composer.json

```diff
@@ -1,14 +1,26 @@
 {
-	"name": "scrivo/highlight.php",
-	"description": "Server side syntax highlighter that supports over 90 languages. It's a PHP port of highlight.js",
-	"keywords": ["code", "syntax", "highlight", "highlight.js", "highlight.php"],
-	"type": "library",
-	"license": "BSD-3-Clause",
-	"support": {
-		"issues": "https://github.com/scrivo/highlight.php/issues",
-		"source": "https://github.com/scrivo/highlight.php"
-	},
-	"autoload": {
-		"psr-0": { "Highlight\\": "" }
-	}
+  "name": "scrivo/highlight.php",
+  "description": "Server side syntax highlighter that supports 176 languages.",
+  "keywords": ["code", "syntax", "highlight", "highlight.js", "highlight.php"],
+  "type": "library",
+  "license": "BSD-3-Clause",
+  "support": {
+    "issues": "https://github.com/scrivo/highlight.php/issues",
+    "source": "https://github.com/scrivo/highlight.php"
+  },
+  "autoload": {
+    "psr-0": { "Highlight\\": "" }
+  },
+  "authors": [
+    {
+      "name": "Geert Bergman",
+      "homepage": "http://www.scrivo.org/",
+      "role": "Project Author"
+    },
+    {
+      "name": "Vladimir Jimenez",
```

Comment:
I added myself to the fork I submitted to Packagist, however I shouldn't belong in the official package.

Reply:
oh no, that's fair. my line of thinking was that the project author should decide who to list under "authors" and not just be snuck in there in a PR

Reply:
True, but honestly I thought of it as "suggestion" 🙂

Reply:
that makes sense. sorry if I came off the wrong way 😅

Reply:
🐪 alright!